packages/example/src/components/connector/connectors-setup.ts

```diff
@@ -80,7 +79,7 @@ function environmentToTezosNetwork(environment: RaribleSdkEnvironment) {
 				network: TezosNetwork.MAINNET
 			}
 		case "dev":
```

Comment:
Please, add node https://dev-tezos-node.rarible.org only for development tezos network